tests/test_stage.py

```diff
@@ -408,6 +408,35 @@ def __init__(self, *, getiter=None, getitem=None, delete=None, post=None):
     assert "@gvanrossum" in comment_body
     assert "not-core-dev" not in comment_body
 
+    # All is right with the Monty Python world.
```

Comment:
Isn’t this a different unit test?

Reply:
If you want to get technical, sure. 😉 But there's a lot of reused setup code so it's not an isolated test in terms of actual code. Plus the whole bot is stateless by design so this won't influence outcomes by having this here.

Reply:
Thanks for explaining!  Good to know if I contribute to or fork your bot :)

Reply:
I have no problem if someone wanted to do a better separation of test code, I just wanted to get this done sooner rather than later so I didn't take the time to extract common code for a refactoring. 😄